collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2862,11 +2838,8 @@ public boolean onDown(MotionEvent e) {
     public boolean onFling(MotionEvent e1, MotionEvent e2, float velocityX,
                            float velocityY) {
         // only check the swipe if it's enabled in preferences
-        SharedPreferences sharedPreferences = PreferenceManager
-                .getDefaultSharedPreferences(this);
-        String navigation = sharedPreferences.getString(
-                PreferenceKeys.KEY_NAVIGATION,
-                PreferenceKeys.NAVIGATION_SWIPE);
+        String navigation = (String) GeneralSharedPreferences.getInstance()
+                .get(PreferenceKeys.KEY_NAVIGATION);
```

Comment:
This means that the default will now be `""` instead of `"swipe"`. I think that means that on new installs with default settings it won't be possible to swipe, right?

Reply:
The default value for `KEY_NAVIGATION` is stored [here](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceKeys.java#L94) and the currently stored value for this key is obtained from [here](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/preferences/GeneralSharedPreferences.java#L46)

So, the default for this preference is still `"swipe"`

Reply:
Ah yes, thanks!

Reply:
Right, I was looking at `AdminSharedPreferences` for some reason, that's why I was thinking the default was `""`.